### PR TITLE
Point our SECURITY.md to the new hackerone program

### PR DESCRIPTION
**Pull request by mcollina**
I have updated the SECURITY.md to point to our brand new program: https://hackerone.com/fastify.

#### Checklist

- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -1,33 +1,111 @@
...
 
-Individuals who find potential vulnerabilities in Fastify are invited to complete a vulnerability report via the dedicated HackerOne tool for Node.js modules: [https://hackerone.com/nodejs-ecosystem](https://hackerone.com/nodejs-ecosystem).
+Individuals who find potential vulnerabilities in Fastify are invited
+to complete a vulnerability report via the dedicated HackerOne page:
+[https://hackerone.com/fastify](https://hackerone.com/fastify).
+
+### Strict measures when reporting vulnerabilities
 
-### How to report a vulnerability
+It is of the outmost importance that you read carefully and follow these
```
Comment (jsumners) [inline SECURITY.md:14]:
```suggestion
It is of the utmost importance that you read carefully and follow these
```

File: SECURITY.md [hunk 2]
```diff
@@ -1,33 +1,111 @@
...
+guidelines to ensure the ecosystem as a whole isn't disrupted due to
+mistakenly reported vulnerabilities:
```
Comment (jsumners) [inline SECURITY.md:16]:
```suggestion
improperly reported vulnerabilities:
```

File: SECURITY.md [hunk 3]
```diff
@@ -1,33 +1,111 @@
...
 
-It is of the utmost importance that you read carefully [**HOW TO REPORT A VULNERABILITY**](https://github.com/nodejs/security-wg/blob/HEAD/processes/third_party_vuln_process.md) written by the Security Working Group of Node.js.
+* Avoid creating new "informative" reports on HackerOne. Only create new
+  HackerOne reports on a vulnerability if you are absolutely sure this
+  should be tagged as an actual vulnerability. Third-party vendors and
+  individual are tracking any new vulnerabilities reported in HackerOne
```
Comment (jsumners) [inline SECURITY.md:21]:
```suggestion
  individuals are tracking any new vulnerabilities reported in HackerOne
```

**Review by jsumners — APPROVED** [review]
LGTM

**Review by makuzaverite — APPROVED** [review]
LGTM

**Review by makuzaverite — APPROVED** [review]

File: SECURITY.md [hunk 4]
```diff
@@ -1,33 +1,111 @@
...
+
+**Severity**: Vulnerability severity is assessed using [CVSS v.3](https://www.first.org/cvss/user-guide).
+More information can be found on [HackerOne documentation](https://docs.hackerone.com/hackers/severity.html)
+
+If a patch is being actively developed by the package maintainer, an additional delay
+can be added with the approval of the security team and the individual who
+reported the vulnerability. 
+
+At this point, a CVE should be requested through the HackerOne platform through
+the UI which should include the Report ID and a summary.
```
Comment (Fdawgs) [inline SECURITY.md:90]:
```suggestion
the UI, which should include the Report ID and a summary.
```

File: SECURITY.md [hunk 5]
```diff
@@ -1,33 +1,111 @@
...
+### Publication
+
+**Delay:** 90 days
+
+Within 90 days after the triage date, the vulnerability must be made public.
+
+**Severity**: Vulnerability severity is assessed using [CVSS v.3](https://www.first.org/cvss/user-guide).
+More information can be found on [HackerOne documentation](https://docs.hackerone.com/hackers/severity.html)
+
+If a patch is being actively developed by the package maintainer, an additional delay
```
Comment (Fdawgs) [inline SECURITY.md:85]:
```suggestion
If the package maintainer is actively developing a patch, an additional delay
```
Improve sentence flow/readability

File: SECURITY.md [hunk 6]
```diff
@@ -1,33 +1,111 @@
...
+improperly reported vulnerabilities:
 
-It is of the utmost importance that you read carefully [**HOW TO REPORT A VULNERABILITY**](https://github.com/nodejs/security-wg/blob/HEAD/processes/third_party_vuln_process.md) written by the Security Working Group of Node.js.
+* Avoid creating new "informative" reports on HackerOne. Only create new
+  HackerOne reports on a vulnerability if you are absolutely sure this
+  should be tagged as an actual vulnerability. Third-party vendors and
+  individuals are tracking any new vulnerabilities reported in HackerOne
+  and will flag them as such for their customers (think about snyk, npm audit, ...).
+* HackerOne reports should never be created and triaged by the same person.
+  If you are creating a HackeOne report for a vulnerability that you found,
```
Comment (Fdawgs) [inline SECURITY.md:24]:
```suggestion
  If you are creating a HackerOne report for a vulnerability that you found,
```

**Review by delvedor — APPROVED** [review]
LGTM

File: SECURITY.md [hunk 7]
```diff
@@ -1,33 +1,111 @@
...
 on the team completely private to the team. This includes agreeing to not notify anyone outside the
 team of issues that have not yet been disclosed publicly, including the existence of issues,
 expectations of upcoming releases, and patching of any issues other than in the process of their work
 as a member of the Fastify Core team.
+
+### Members
+
+* [__Matteo Collina__](https://github.com/mcollina), <https://twitter.com/matteocollina>, <https://www.npmjs.com/~matteo.collina>
+* [__Tomas Della Vedova__](https://github.com/delvedor), <https://twitter.com/delvedor>, <https://www.npmjs.com/~delvedor>
+* [__Vincent Le Goff__](https://github.com/zekth)
```
Comment (climba03003) [inline SECURITY.md:110]:
```suggestion
* [__Vincent Le Goff__](https://github.com/zekth)
* [__KaKa Ng__](https://github.com/climba03003)
```

**Review by zekth — APPROVED** [review]
lgtm

**Pull request merged**